docs/pages/toolchain/katana/genesis.md

```diff
@@ -149,11 +155,11 @@ The genesis config file is a JSON file that contains the following fields:
 	"classes": [
 		{
 			"class": "path/to/file/erc20.json",
-			"classHash": "0x8"
+			"name": "MyClass"
 		},
 		{
 			"class": "path/to/file/universal_deployer.json",
-			"classHash": "0x444"
+			"classHash": "0x444" //or ("name": "className")
```

Comment:
not sure we need the comment with the alternative, as we already have an example of `"name": "MyClass"`.

Reply:
We could remove other comments too for accountName, contractName and feeToken class wdyt @kariy?